### PR TITLE
Improve the authentication mechanism

### PR DESCRIPTION
**Pull request by dunglas**
By passing top properties to the `Admin` component. Docs included.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,6 +114,87 @@ Any [field](https://marmelab.com/admin-on-rest/Fields.html) or [input](https://m
...
+    options.headers = new Headers({ Accept: 'application/ld+json' });
+  }
+
+  options.headers.set('Authorization', `Bearer ${localStorage.getItem('token')}`);
+  return fetchHydra(url, options);
+};
+
+class Admin extends Component {
+  render() {
+    return <HydraAdmin restClient={hydraClient(entrypoint, fetchWithAuth)}, authClient={authClient}/>
```
Comment (Simperfit) [inline README.md]: should be `<HydraAdmin restClient={hydraClient(entrypoint, fetchWithAuth)} authClient={authClient}/>`

Comment (Simperfit) [inline README.md]: entrypoint is required: 
```javascript
<HydraAdmin restClient={hydraClient(entrypoint, fetchWithAuth)} authClient={authClient} entrypoint={entrypoint} />
```

File: README.md [hunk 2]
```diff
@@ -114,6 +114,87 @@ Any [field](https://marmelab.com/admin-on-rest/Fields.html) or [input](https://m
...
+    options.headers = new Headers({ Accept: 'application/ld+json' });
+  }
+
+  options.headers.set('Authorization', `Bearer ${localStorage.getItem('token')}`);
+  return fetchHydra(url, options);
+};
+
+class Admin extends Component {
+  render() {
+    return <HydraAdmin restClient={hydraClient(entrypoint, fetchWithAuth)} authClient={authClient}/>
```
Comment (Simperfit) [inline README.md]: entrypoint is required :

Comment (dunglas) [inline README.md]: Good catch, I've just made it optional.

**Pull request merged**